livy/src/main/resources/interpreter-setting.json

```diff
@@ -227,5 +227,21 @@
       "editOnDblClick": false,
       "completionKey": "TAB"
     }
+  },
+  {
+    "group": "livy",
+    "name": "shared",
+    "className": "org.apache.zeppelin.livy.LivySharedInterpreter",
```

Comment:
Will this be automatically added to `interpreter.json`, or the user has to add it manually? And if this class is not present in `interpreter.json` (by default), will the previous interpreter works?

Reply:
It will be added to interpreter.json automatically.